[12.0][FIX] Remove unnecessary utf8 coding comments and replace http license links

skeller1 opened the pull request:
**Remove linting error `C8202(unnecessary-utf8-coding-comment)` for 12.0 pipeline**


![utf8_comments](https://user-images.githubusercontent.com/226753/81699459-ad676c00-9467-11ea-9305-73bb3dd3aa7a.png)

**Replace old http license links with https**

NL66278 commented on crm_location_nuts/models/__init__.py:
```diff
@@ -1,3 +1,2 @@
-# -*- coding: utf-8 -*-
 
```
You should remove the blank lines remaining at the beginning of this file (and some others) as well (or replace them with a license line - my personal preference).

NL66278 commented on crm_location_nuts/tests/test_crm_location_nuts.py:
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Tecnativa - David Vidal
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
```
And when you are at it anyway: change the license lines to https:

skeller1 replied:
Did it, thank you

NL66278 reviewed (changes requested):
:+1: Thanks for taking up this legacy stuff. With a few extra changes it will be perfect.

skeller1 committed:
[FIX] make license links secure and remove utf8 encoding comments

skeller1 reviewed (commented):
Ready to go

skeller1 reviewed (commented):
Changed all license links and removed utf-8 encoding comments from 12.0

NL66278 commented on crm_lead_code/__manifest__.py:
```diff
@@ -15,7 +15,7 @@
 #    GNU Affero General Public License for more details.
```
For this file replace the whole comment block at the beginning with the shortened license and copyright lines.

NL66278 commented on crm_lead_firstname/__manifest__.py:
```diff
@@ -1,5 +1,5 @@
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
And replace Copyright sign with the word `Copyright`.

NL66278 commented on crm_lead_firstname/models/crm_lead.py:
```diff
@@ -1,5 +1,5 @@
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
Replace copyright sign with word.

NL66278 commented on crm_lead_firstname/tests/test_crm_lead.py:
```diff
@@ -1,5 +1,5 @@
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
And here and everywhere: Replace copyright sign with word.

NL66278 commented on crm_lead_vat/tests/__init__.py:
```diff
@@ -1,4 +1,4 @@
 # Copyright 2015 Antiun Ingeniería, S.L.
```
There should be no copyright lines in __init__.py files at all.

NL66278 reviewed (changes requested):
:+1: A great cleanup! But now that more files are changed, there are a small number of additional things todo.

skeller1 commented:
@NL66278:

- [x] Removed all copyright comments from init files
- [x] Cleaned up copyright sign

Question:

1. Could I change that line:
`# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).`
to
`# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).`

2. Could I change: `# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
` to `# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).`

NL66278 commented:
@skeller1 I think all those three lines are valid (and equivalent). I mostly use the agpl.html (without -3.0 but with html), but the shortest version would maybe even be better.

NL66278 reviewed (approved):
:+1: Thank you for all the work!

simahawk commented:
/ocabot merge nobump
